suite-native/module-accounts/src/components/AccountListItem.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { TouchableOpacity } from 'react-native';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { AccountsRootState, selectAccountName } from '@suite-common/wallet-core';
+import { Box, Text } from '@suite-native/atoms';
+import {
+    AccountsStackParamList,
+    AccountsStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { Account } from '@suite-common/wallet-types';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { CryptoIcon } from '@trezor/icons';
+
+type AccountListItemProps = {
+    account: Account;
+};
+
+const accountListItemStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.gray0,
+    padding: utils.spacings.medium,
+    borderRadius: 12,
+    marginBottom: utils.spacings.small,
+}));
+
+export const AccountListItem = ({ account }: AccountListItemProps) => {
+    const navigation =
+        useNavigation<StackNavigationProps<AccountsStackParamList, AccountsStackRoutes.Accounts>>();
+    const { applyStyle } = useNativeStyles();
+    const accountName = useSelector((state: AccountsRootState) =>
+        // eslint-disable-next-line no-restricted-syntax
+        selectAccountName(state, account.key),
```

Comment:
What about making it like this? `const  selectAccountName = (key) => (state) => {use state & key }`

Reply:
This is normal pattern recommended in the documentation of reselect. I would honestly just stick with this. Do you think the approach recommended by you has any benefits?

https://github.com/reduxjs/reselect#q-how-do-i-create-a-selector-that-takes-an-argument

Reply:
I agree with @dahaca that it looks much better because you can do `useSelector(selectAccountName(key))` without need to pass `state` arg directly, it will break memoization because it will create new instance of selector functions with every rerender, this could be solved only by memoizing it with `useMemo` and it's more complicated in the end. Let's stick with reselect recommended approach.

Reply:
Didn't realise, thanks :)